embark-ui/src/components/Console.js

```diff
@@ -123,6 +148,10 @@ class Console extends Component {
                     this.handleChange(e.target.value, () => {
                       this.handleSubmit(e);
                     });
+                  } else if(e.keyCode === 38) {
+                    this.moveHistoryUp();
+                  } else if(e.keyCode === 40) {
+                    this.moveHistoryDown();
                   }
```

Comment:
I think we should add a binding to `esc` too to clear the field.

Reply:
Agree, will do